keycloak/user-storage-provider/pom.xml

```diff
@@ -20,8 +20,8 @@
 
     <properties>
         <version.compiler.maven.plugin>3.5.1</version.compiler.maven.plugin>
```

Comment:
Remove this line as well, versions should be set on super pom

Reply:
Done